Resources/Prototypes/_NF/Recipes/Crafting/Graphs/magic_crayon.yml

```diff
@@ -0,0 +1,28 @@
+- type: constructionGraph
+  id: CrayonMagic
```

Comment:
Change to `magic_crayon` to match the crafting recipe.

Reply:
Still not there

Reply:
I didn't understand your comment.

Reply:
Wait feck I looked at the wrong file